examples/kubernetes-istio/cilium-kube-inject.sed

```diff
@@ -0,0 +1,13 @@
+# Inject Cilium's /var/run/cilium directory into each Istio sidecar proxy to
```

Comment:
Nit: I think `mount` is the correct word and not `inject` a directory?

Reply:
I prefer to use the "inject" term because this is in the context of running the `istioctl kube-inject` commands, which modifies the deployment specs to _inject_ new containers, labels, etc. So "inject" seems like the proper term.